### PR TITLE
feat: ty page badge animations

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Animations added to TY page badges
- Both cases approved by Nathan
- Reqs: https://kiva.atlassian.net/browse/MP-1013

Video to check: https://www.loom.com/share/6379316d598b41eea0435f269f3a4016?sid=6bb80753-fbd2-4c8c-96c1-6c7cc6bce924

### CODE REVIEW

File: src/components/MyKiva/BadgeContainer.vue [hunk 1]
```diff
@@ -150,12 +158,19 @@ const handleBadgeClick = () => {
...
+watch(() => props.showShine, newValue => {
+	if (newValue) {
+		handleShine();
 	}
-};
+});
+
+onMounted(() => {
+	if (props.showShine) {
+		handleShine();
```
Comment (dyersituations) [inline src/components/MyKiva/BadgeContainer.vue:171]: Could this be related with `{ immediate: true }` on the `watch`?

File: src/components/Thanks/MyKiva/ThanksBadges.vue [hunk 2]
```diff
@@ -382,4 +446,162 @@ watch(() => badgeData.value, () => {
...
+/** Rays */
+.ray_box {
+	@apply tw-absolute tw-mx-auto tw-left-0 tw-right-0 tw-bottom-0 tw-aspect-square tw-top-5;
+	width: 250px;
+}
+
+.ray {
+	background: linear-gradient(180deg, rgba(241, 179, 67, 0.03) 0%,
+		rgba(241, 179, 67, 0.25) 50%, rgba(241, 179, 67, 0.03) 100%);
+    border-radius:80% 80% 0 0;
```
Comment (dyersituations) [inline src/components/Thanks/MyKiva/ThanksBadges.vue:459]: Fix indentation issues throughout these new styles.

File: src/components/MyKiva/BadgeContainer.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/components/MyKiva/BadgeContainer.vue:171]: Already tested and doesn't work...

File: src/components/Thanks/MyKiva/ThanksBadges.vue (hunk 2 above)
Comment (dyersituations) [reply src/components/Thanks/MyKiva/ThanksBadges.vue]: @roger-in-kiva I saw your latest commit, there are indentation issues throughout this style block.

File: src/components/Thanks/MyKiva/ThanksBadges.vue [hunk 3]
```diff
@@ -382,4 +446,175 @@ watch(() => badgeData.value, () => {
...
+		top: 25%;
+		left: -20%;
+	}
+}
+
+.ray9 {
+	transform: rotate(120deg);
+	animation-delay: 2s;
+	height: 80px;
+    width: 10px;
```
Comment (dyersituations) [inline src/components/Thanks/MyKiva/ThanksBadges.vue:579]: Looks like we're still seeing a lot of indentation differences.

Comment (roger-in-kiva) [reply src/components/Thanks/MyKiva/ThanksBadges.vue:579]: Weird, I don't see that and no lint issues about it.
Screenshot 2024-12-02 at 2 40 18 p m

Comment (dyersituations) [reply src/components/Thanks/MyKiva/ThanksBadges.vue:579]: Right, it has to do with your VS Code settings. You can see the whitespace differences here in GitHub.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**